security/RememberLoginHash.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+class RememberLoginHash extends DataObject {
```

Comment:
Should note that `DeviceID` is transitory - you won't get the same ID after logging out explicitly, and then using auto-login on the same device again.